functional-test/src/test/groovy/org/rundeck/util/container/BaseContainer.groovy

```diff
@@ -310,6 +310,20 @@ abstract class BaseContainer extends Specification implements ClientProvider {
         }
     }
 
+    void disableScheduledAndDeleteProject(String projectName) {
```

Comment:
it would be better that the properties were received by parameter. This way  we could set any property instead of just this 2 properties

Reply:
Done